docker-bake.hcl

```
@@ -0,0 +1,148 @@
+// TODO handle latest / latest-variant / latest-lts
+
+group "linux" {
+  targets = [
+    "alpine_jdk8",
+    "centos7_jdk8",
+    "centos8_jdk8",
+    "debian_jdk8",
+    "debian_jdk11",
+    "debian_slim_jdk8",
+  ]
+}
+
+group "linux-arm64" {
+  targets = [
+    "centos8_jdk8",
+    "debian_jdk8",
+    "debian_jdk11",
+    "debian_slim_jdk8",
+  ]
+}
+
+group "linux-s390x" {
+  targets = [
+    "debian_jdk8",
+    "debian_jdk11",
+    "debian_slim_jdk8",
+  ]
+}
+
+group "linux-ppc64le" {
+  targets = [
+    "centos8_jdk8",
+    "debian_jdk8",
+    "debian_jdk11",
+    "debian_slim_jdk8",
+  ]
+}
+
+group "windows" {
+  targets = [
+    "windows_1809_jdk11",
+    "windows_2019_jdk11",
+  ]
+}
+
+variable "JENKINS_VERSION" {
+  default = "2.300"
+}
+
+variable "JENKINS_SHA" {
+  default = "2f6aa548373b038af4fb6a4d6eaa5d13679510008f1712532732bf77c55b9670"
+}
+
+variable "REGISTRY" {
+  default = "docker.io"
+}
+
+variable "OWNER" {
+  default = "jenkins"
+}
+
+target "alpine_jdk8" {
+  dockerfile = "8/alpine/hotspot/Dockerfile"
+  context = "."
+  args = {
+    JENKINS_VERSION = JENKINS_VERSION
+    JENKINS_SHA = JENKINS_SHA
+  }
+  tags = ["${REGISTRY}/${OWNER}/jenkins:${JENKINS_VERSION}-alpine"]
+  platforms = ["linux/amd64"]
+}
+
+target "centos7_jdk8" {
+  dockerfile = "8/centos/centos7/hotspot/Dockerfile"
+  context = "."
+  args = {
+    JENKINS_VERSION = JENKINS_VERSION
+    JENKINS_SHA = JENKINS_SHA
+  }
+  tags = ["${REGISTRY}/${OWNER}/jenkins:${JENKINS_VERSION}-centos7"]
+  platforms = ["linux/amd64"]
+}
+
+target "centos8_jdk8" {
+  dockerfile = "8/centos/centos8/hotspot/Dockerfile"
+  context = "."
+  args = {
+    JENKINS_VERSION = JENKINS_VERSION
+    JENKINS_SHA = JENKINS_SHA
+  }
+  tags = ["${REGISTRY}/${OWNER}/jenkins:${JENKINS_VERSION}-centos"]
+  platforms = ["linux/amd64", "linux/ppc64le", "linux/arm64"]
+}
+
+target "debian_jdk8" {
+  dockerfile = "8/debian/buster/hotspot/Dockerfile"
+  context = "."
+  args = {
+    JENKINS_VERSION = JENKINS_VERSION
+    JENKINS_SHA = JENKINS_SHA
+  }
+  tags = ["${REGISTRY}/${OWNER}/jenkins:${JENKINS_VERSION}"]
+  platforms = ["linux/amd64", "linux/ppc64le", "linux/arm64", "linux/s390x"]
```

The JDK 8 on s390x requires openJ9 for acceptable performance.  I'd rather not complicate our environment by using an OpenJ9 on s390x.  I think we should accept that s390x will only be Java 11.
```suggestion
  platforms = ["linux/amd64", "linux/ppc64le", "linux/arm64"]
```